install_config/adding_hosts_to_installs.adoc

```diff
@@ -0,0 +1,195 @@
+[[install-config-adding-hosts-to-installs]]
+= Adding Hosts to an Existing Install
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+Depending on how your {product-title} cluster was installed, you can add new
+hosts (either nodes or masters) to your installation by using the install tool
+for quick installations, or by using the *_scaleup.yml_* playbook for advanced
+installations.
+
+[[adding-nodes-or-reinstalling-quick]]
+== Adding Hosts Using the Quick Installer Tool
+
+If you used the quick install tool to install your {product-title} cluster, you
+can use the quick install tool to add a new node host to your existing cluster,
+or to reinstall the cluster entirely.
+
+[NOTE]
+====
+Currently, you can not add new master hosts if you used the quick install. 
```

Comment:
This makes it sound like you can never add new masters if you started w/ the quick install (which you could, if you instead switched to using the advanced method). I assume this is more specifically trying to say that you can't use the quick installer tool to add new masters.